### PR TITLE
🌙 Add closeVessel fees tests

### PR DESCRIPTION
**Pull request by MiksuJak**

### CODE REVIEW

File: test/trinity/BorrowerOperations_FeesTest.js [hunk 1]
```diff
@@ -196,6 +207,43 @@ contract("BorrowerOperations_Fees", async accounts => {
...
+            await borrowerOperations.collectVesselFee(erc20.address, alice)
+            
+            const treasuryBalanceBefore = await debtToken.balanceOf(treasury)
+
+            await mintDebtTokens(alice, vesselTotalDebt)
+            await closeVessel(alice)
+
+            const treasuryBalanceAfter = await debtToken.balanceOf(treasury)
+
+            assert.isTrue(treasuryBalanceBefore.eq(treasuryBalanceAfter))
```
Comment (JustynaBroniszewska) [inline test/trinity/BorrowerOperations_FeesTest.js]: I would prefer us to use assert.equal(A, B), so the errors will be more descriptive and better than: True is not False

Comment (MiksuJak) [reply test/trinity/BorrowerOperations_FeesTest.js]: assert.equal matcher does not work on the BN type that they use

Comment (nezouse) [reply test/trinity/BorrowerOperations_FeesTest.js]: assert.equal(before.toString(), after.toString())

**Review by nezouse — APPROVED** [review]

File: test/trinity/BorrowerOperations_FeesTest.js [hunk 2]
```diff
@@ -196,6 +207,43 @@ contract("BorrowerOperations_Fees", async accounts => {
...
+            
+            const treasuryBalanceBefore = await debtToken.balanceOf(treasury)
+
+            await mintDebtTokens(alice, vesselTotalDebt)
+            await closeVessel(alice)
+
+            const treasuryBalanceAfter = await debtToken.balanceOf(treasury)
+
+            assert.equal(treasuryBalanceBefore.toString(), treasuryBalanceAfter.toString())
+        })
```
Comment (nezouse) [inline test/trinity/BorrowerOperations_FeesTest.js]: Will also pass after splitting treasury into treasury and distributor

**Review by nezouse — APPROVED** [review]

**Review by Szymx95 — APPROVED** [review]

**Pull request merged**